config/webpack/build-aot.webpack.config.js

```diff
@@ -37,7 +41,7 @@ function getWebpackConfig(skyPagesConfig, argv) {
 
     // Disable sourcemaps for production:
     // https://webpack.js.org/configuration/devtool/#production
```

Comment:
This comment no longer makes sense. Do you mind updating the comment to explain why we're including `.map` files with the bundle?